site/schemas/organisation.schema.ts

```diff
@@ -16,6 +17,7 @@ export const defaultModes: ModeType = {
     tram: Datasource.tnds,
     ferryService: Datasource.tnds,
     rail: Datasource.tnds,
+    underground: Datasource.tnds,
 };
 
 export const organisationSchema = z.object({
```

Comment:
On line 27 we can remove `.default(defaultModes)`  and the const `defaultModes` since we are setting defaults individually in `modeSchema` now.

Reply:
done